### PR TITLE
Rewrote chp10 `diagonal_winner` exercise to use nested lists

### PR DESCRIPTION
**Pull request by spamegg1**
- should we add new hints? I don't think so.

### CODE REVIEW

**Comment by spamegg1** [comment]
Oops... It looks like I forgot to change `def diagonal_winner(row1, row2, row3):` to `def diagonal_winner(board):` in the final text... I shouldn't make a PR for it, could you fix it? Thank you!

**Comment by alexmojaki** [comment]
I notice that you create a new fork each time you make a PR, and delete the previous fork.

You don't need to do that every time. Rather you should be configuring your local repository to be able to pull the latest changes from my side. See https://gist.github.com/Chaser324/ce0505fbed06b947d962 or https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork and https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

Basically:

```
git remote add upstream https://github.com/alexmojaki/futurecoder.git
git pull upstream master
```
